### DISCUSSION TITLE
Cell Id Binary

### DISCUSSION THREAD

**Discussion by ghost** [aaliddell/s2cell#2] [General]
Is there a way to get consistent Cell Id Binary for each token ? like you did in the documentation ? i tried converting the cell id base 10 but sometimes it gives off a binary length of 65 sometimes 63 so i am wondering is there a method to do this in your library ? like you showed in your documentation or you were converting it with `bin( )` as well 
<html>
<body>

3170534137668829184 | 0010110000000000000000000000000000000000000000000000000000000000 | 1 | 2c
-- | -- | -- | --

</body>
</html>

**Comment by aaliddell** [comment]
I think I just did it with a format string along the lines of:

format(cell_id, ‘064b’)

That should give you at least 64 chars of binary string with zero padding.
